spec/helpstation/processors/by_key_fetcher_spec.rb

```diff
@@ -22,7 +22,7 @@
       expect(subject).to be_a(Substation::Response::Success)
       expect(subject.output).to eq(
         my_input: input[:my_input],
-        my_output: output
+        my_output: 'test and {:mock_service=>:mock_value}'
```

Comment:
I actually prefer writing it out. Otherwise if the `env` is nil for some reason then you might be doing `nil == nil` check and it will pass silently.